test/api.js

```diff
@@ -1516,6 +1516,141 @@ describe('/v1', function() {
         });
       });
 
+      it('incremental authorization with include_granted_scopes', function() {
+        mockAssertion().reply(200, VERIFY_GOOD);
+        return Server.api.post({
+          url: '/authorization',
+          payload: authParams({
+            scope: 'foo bar bar',
+            access_type: 'offline'
+          })
+        }).then(function(res) {
+          assert.equal(res.statusCode, 200);
+          assertSecurityHeaders(res);
+          return Server.api.post({
+            url: '/token',
+            payload: {
+              client_id: clientId,
+              client_secret: secret,
+              code: res.result.code,
+            }
+          });
+        }).then(function(res) {
+          assert.equal(res.statusCode, 200);
+          assertSecurityHeaders(res);
+          assert.equal(res.result.token_type, 'bearer');
+          assert(res.result.access_token);
+          assert(res.result.refresh_token);
+          assert.equal(res.result.access_token.length,
+            config.get('unique.token') * 2);
+          assert.equal(res.result.refresh_token.length,
+            config.get('unique.token') * 2);
```

Comment:
FWIW I don't think these detailed "is the token in the expected format?" tests are necessary to duplicate here, that should be well covered by existing tests.